src/main/java/org/example/models/JobRoleFilteredRequest.java

```diff
@@ -0,0 +1,88 @@
+package org.example.models;
+
+import io.swagger.annotations.ApiModelProperty;
+
+import javax.ws.rs.QueryParam;
+import java.sql.Date;
+import java.util.List;
+
+public class JobRoleFilteredRequest {
+    @ApiModelProperty(
+            value = "Job role's name",
+            required = true,
+            example = "Delivery manager")
+    @QueryParam("roleName")
+    private String roleName;
+
+    @ApiModelProperty(
+            value = "Role's location, defined by ENUM value",
+            required = true,
+            example = "[Gdańsk]")
```

Comment:
defined by ENUM value: I would advise Gdansk ;) How UK guys are supposed to write ń :D